model/pipeline/src/main/proto/beam_runner_api.proto

```diff
@@ -217,12 +217,12 @@ message ExecutableStagePayload {
   // PTransform the ExecutableStagePayload is the payload of.
   string input = 2;
 
-  // Side Input PCollection ids. Each must be present as a value in the inputs of
-  // any PTransform the ExecutableStagePayload is the payload of.
-  repeated string side_inputs = 3;
+  // The side inputs required for this executable stage. Each must be present as a side input of
+  // exactly one PTransform within this ExecutableStagePayload.
```

Comment:
I would modify this spec a little - "Each Side Input of each PTransform within this ExecutableStagePayload must be represented within this field." or thereabouts.

That way it represents the minimum contents instead of the maximum contents; if we make more side inputs available than required, I don't expect either harness to break, just to be very confused.

Reply:
Done.